include/rabbit.hrl

```diff
@@ -77,7 +77,8 @@
           gm_pids,                     %% transient
           decorators,                  %% transient, recalculated as above
           state,                       %% durable (have we crashed?)
-          policy_version}).
+          policy_version,
+          pending_slave_pids}).
```

Comment:
@dcorbacho I believe `amqqueue.slave_pids_pending_shutdown` would be clearer. WDYT?